homeworks/Vladimir.Karbivnychiy_VladimirSkarb/5-rxjs/index.html

```diff
@@ -0,0 +1,103 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+<section class="post-section">
+    <header class="header" data-header>
```

Comment:
There's no animation between **"Get an amazing discount BUY NOW"** and **"Logo BUY NOW"**. In the example that Taras has provided, your old **"Get an amazing discount BUY NOW"** header should hide, and **"Logo BUY NOW"** header should appear.

You can find an example of how it should work here at 0:20 - 0:30
https://www.dropbox.com/s/5t29uz3yvkqox3a/advanced-task.mov?dl=0

I suggest you create each header separately and then just hide/show needed 😄